mcc/odin/odin/snapshot_list.go

```diff
@@ -0,0 +1,18 @@
+package odin
+
+import (
+	"github.com/aws/aws-sdk-go/service/rds"
+	"github.com/aws/aws-sdk-go/service/rds/rdsiface"
+)
+
+// ListSnapshots return a list of all DBSnapshots.
+func ListSnapshots(
+	instanceName string,
+	svc rdsiface.RDSAPI,
```

Comment:
The mantra with Go is to maintain as much context as possible together. This allows to use shorthand variables in internal implementations when its function is "obvious". The hardest part is determining when obvious for you will be obvious for others 😃. It's recommended to be used only in internal variables and never on the function signature, so we may be in the wrong here. Anyway, AWS SDK examples use `svc` as shorthand for service in their own examples.

Another mantra is implicit convention and that's why testing library handlers are always named `t` (and benchmark ones are named `b`).